src/test/java/com/google/sps/UtilityTest.java

```diff
@@ -115,4 +122,50 @@ public void nullValue() {
 
     Assert.assertEquals(actual, expected);
   }
+
+  /** 
+   * Tests for buildQuestion function.
+   * Testing for null input is out of scope since it never happens.
+   */
+  @Test
+  public void normalResult() {
+    ResultSet testResultTest = mock(ResultSet.class);
```

Comment:
Why is test twice in the name? Maybe you meant `testResultSet`?

Reply:
oh yeah ahahahah little typo